bot/src/wrapper/_communication.py

```diff
@@ -208,3 +211,60 @@ async def caller(session: aiohttp.ClientSession) -> ShopBuyOutput:
                 raise UnknownNetworkError(message)
 
         return await make_request(session, caller)
+
+
+class UserRawAPI:
+    """A bundle of API available to use for user endpoint."""
+
+    @staticmethod
+    async def get_user(session: aiohttp.ClientSession, user_id: int) -> UserIdGetOutput:
+        """Get the User information by user id through direct HTTP request."""
+
+        async def caller(session: aiohttp.ClientSession) -> UserIdGetOutput:
+            async with session.get(f"user/{user_id}") as resp:
+                if resp.ok:
+                    return await resp.json()
+                if resp.status == Status.NOT_FOUND:
+                    message = f"User with user id {user_id} cannot be found"
+                    raise DoNotExistError(message)
+                message = f"Undefined behaviour bot.src.wrapper.UserRawAPI.get_user, Status received {resp.status}"
+                raise UnknownNetworkError(message)
+
+        return await make_request(session, caller)
+
+    @staticmethod
+    async def create_user(session: aiohttp.ClientSession, user_id: int) -> None:
+        """Create the user by user id through a direct HTTP request."""
+
+        async def caller(session: aiohttp.ClientSession) -> None:
+            async with session.get(f"user/{user_id}") as resp:
```

Comment:
Creating a user requires a POST request

Reply:
Mb, forgot to change it when copy and paste